java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -27,6 +27,8 @@ public interface TemplateConstants {
   String SPANNER_GCS_OUTPUT_GCS_PATH = "spanner.gcs.output.gcs.path";
   String SPANNER_GCS_OUTPUT_GCS_SAVEMODE = "spanner.gcs.output.gcs.saveMode";
   String SPANNER_GCS_OUTPUT_FORMAT = "spanner.gcs.output.gcs.format";
+  String SPANNER_GCS_TEMP_TABLE = "spanner.gcs.temp.table";
```

Comment:
missing corresponding updates in templates.properties file

Reply:
Done.